### ISSUE TITLE
[C3M0_B] - Raubgold

### ISSUE THREAD

**Issue by MuadDib90** [Realm667/WolfenDoom#999]
1. After reaching the exit, game "stops", and you can't move. I reported this a while ago, you can kill yourself with a grenade and when you start shooting, this can provoke some soldiers to shoot at you and switch the alarm on. Then you are a swiss chease. Shouldn't Ryan suppose to be imortal from this point to solve this issue? He is not.
2. When ending screen appears with the text, there should be only thunder, rain and dias at the background, right? For me the music starts again from the begining, like at the start of the game. It disturbs the effect it should have.

**Comment by hawkwind3** [comment]
Yesterday, I played this level with the C3.1.1 release and all went well. Ryan was "captured" at the exit and the 2 page end text displayed in the gestapo projector room. Played on Just Warming Up skill.

**Comment by MuadDib90** [comment]
Yes, all that appeared (was present). But except that, the music started, which shouldn't.

**Comment by MuadDib90** [comment]
Tested again today.
Results:
1. First point seems to be resolved. I couldn't recreate it again today. And I barely remember us talking about it in yet other post and been solved.
2. About the second, well, still persist, unfortunatelly.
3. Not an issue exactly, but rather proposal - small details. I'll add it to the first post as another checkbox.
Text could be: "What are you doing, Ryan? We need to investigate. You can't escape yet."


<details>
<summary>Figure to point 3</summary>

![Screenshot_BoA_20220227_190826](https://user-images.githubusercontent.com/73436978/155894585-7480ee0b-6aed-47ea-8cad-70ccf3c0b234.png)

</details>

@Tormentor667 @AFADoomer

**Comment by Tormentor667** [comment]
@Username-N00b-is-not-available when should 3. appear?
@AFADoomer can you take a look at 2.?
